docs/AA--Javascript-in-EE/components/form/select/editor-datetime-select.md

```diff
@@ -0,0 +1,63 @@
+
+# `<EditorDatetimeSelect />`
+
+**EditorDatetimeSelect** applies WordPress admin formatting to the [DatetimeSelect](datetime-select.md) component by wrapping it with the Gutenberg [BaseControl](https://github.com/WordPress/gutenberg/tree/master/components/base-control) component. This wil override the label applied by the **DatetimeSelect** component in favour of the one applied by the **BaseControl** component. Help text can also be specified which will be displayed after the **DatetimeSelect** component.
+
+The **EditorDatetimeSelect** component accepts **all** of the properties that the the **DatetimeSelect** component does as well as the following:
+
+```jsx
+<EditorDatetimeSelect
+    id="datetime-select-1" 
+    className="datetime-select"
+    help={ 'This text will be displayed after the select input to aid the user in understanding its purpose or effect.' }
+/>
+```
+
+
+## Props
+
+The component accepts the following props:
+
+### id
+
+The id of the element to which labels and help text are being generated. That element should be passed as a child.
+
+- Type: `String`
+- Required: Yes
```

Comment:
This is no longer a handled prop.

Reply:
sigh... that's just because you changed it from `id` to `customId` in the `withBaseControl` HoC... 
how about we just change that back in the HoC instead of having to change everything else?

Reply:
What's with the _sigh_?  `customId` is an _argument_ provided to `withBaseControl` not a prop and that was updated in the documentation for a pull request _you reviewed_!!.  `id` as a prop is _still_ passed through to `<DatetimeSelect />` but the prop _is_not_ handled by `DatetimeEditorSelect` because it's passed as an **argument** to the `withBaseControl` HOC.